src/context/FormContext.tsx

```diff
@@ -168,6 +173,7 @@ export const FormProvider = (props: { children: ReactNode }) => {
         setStatusHasChanged,
         statusChangeEvent,
         updateStatusChangeEvent,
+        ...props.contextValues,
```

Comment:
Yep, that's the plan, you can override the default values but this is only used in a test so it has no impact on the actual form. If I did this first, it wouldn't work because every value has a default which would override everything I'm passing in.

Reply:
we can use the prop value as the initial value for the state. would that work for the test?

Reply:
Discussed offline, now mocking the hook instead of using the provider